src/guc.c

```diff
@@ -121,16 +120,6 @@ _guc_init(void)
 							NULL,
 							assign_max_cached_chunks_per_hypertable_hook,
 							NULL);
-	DefineCustomStringVariable("timescaledb.telemetry_endpoint",
```

Comment:
Ah, this just removes dead code, since this GUC isn't used anywhere right now. Or do you mean you want the user to be able to do `show timescaledb.telemetry_endpoint` (didn't realize that was what the GUC was still there for).

Reply:
This GUC is marked  `PGC_INTERNAL`, so it has always been there only for the `SHOW` command.

Yes, we happened to read from it before, and now we use the macros the GUC is set from. But what is the difference?

I actually don't mind removing the GUC since it is not settable. But I heard arguments for this internal GUC before, so that's why I am wondering what changed?

FWIW, I prefer a settable GUC, but that train has left the station.

Reply:
Yeah, I do remember talking about the GUC being internal, but I don't remember anymore why this had to do with others wanting to take out the URI parsing. My impression from that conversion was that because we were remove parsing, we would replace one of the macros with the GUC (like I originally had a long time ago). But now that I think about it, that's silly, since you have 3 such macros.
Conclusion: I'm totally fine with removing the GUC, and I don't remember anymore why it was kept around.